src/Umbraco.Web/Telemetry/ReportSiteTask.cs

```diff
@@ -0,0 +1,133 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Umbraco.Core;
+using Umbraco.Core.Configuration;
+using Umbraco.Core.IO;
+using Umbraco.Core.Logging;
+using Umbraco.Web.Scheduling;
+
+namespace Umbraco.Web.Telemetry
+{
+    public class ReportSiteTask : RecurringTaskBase
+    {
+        private IRuntimeState _runtime;
+        private IProfilingLogger _logger;
+        private static HttpClient _httpClient;
+
+        public ReportSiteTask(IBackgroundTaskRunner<RecurringTaskBase> runner, int delayBeforeWeStart, int howOftenWeRepeat, IRuntimeState runtime, IProfilingLogger logger)
+            : base(runner, delayBeforeWeStart, howOftenWeRepeat)
+        {
+            _runtime = runtime;
+            _logger = logger;
+            _httpClient = new HttpClient();
+        }
```

Comment:
Now this actually overrides the static _httpClient if you created a new ReportSiteTask. This this should just be initialized outside of ctor
```suggestion
        private static HttpClient _httpClient = new HttpClient();

        public ReportSiteTask(IBackgroundTaskRunner<RecurringTaskBase> runner, int delayBeforeWeStart, int howOftenWeRepeat, IRuntimeState runtime, IProfilingLogger logger)
            : base(runner, delayBeforeWeStart, howOftenWeRepeat)
        {
            _runtime = runtime;
            _logger = logger;
        }
```